web/src/lib/components/Login.svelte

```diff
@@ -42,7 +42,7 @@
 <div class="pt-10 flex justify-center items-center">
     {#if qr}
         <div>
-            <p class="mb-4">Scan with a <a class="link" href="https://github.com/fiatjaf/lnurl-rfc#lnurl-documents" target="_blank">compatible</a> wallet to enter the marketplace.</p>
+            <p class="mb-4 text-center">Scan with <a class="link" href="https://breez.technology/">Breez,</a> <a class="link" href="https://phoenix.acinq.co/">Phoenix,</a> <a class="link" href="https://thunderhub.io/">Thunderhub,</a> <a class="link" href="https://zeusln.app/">Zeus,</a> <a class="link" href="https://getalby.com/">Alby,</a> or any <a class="link" href="https://github.com/fiatjaf/lnurl-rfc#lnurl-documents" target="_blank">LNurl compatible</a> wallet to enter the marketplace.</p>
```

Comment:
Can one "scan" with Thunderhub though? Isn't it a web app?

Reply:
Addressed this with [befad84](https://github.com/PlebeianTech/plebeian-market/pull/12/commits/befad84d89ffda42e09a66ab0ae7335ba0b3adab) but we may want to look at it some more, it is getting a bit wordy this way.